joern-cli/frontends/jssrc2cpg/src/main/scala/io/joern/jssrc2cpg/astcreation/AstForDeclarationsCreator.scala

```diff
@@ -146,7 +146,7 @@ trait AstForDeclarationsCreator { this: AstCreator =>
     val specifiers = declaration
       .json("specifiers")
       .arr
-      .toSeq
+      .toList
```

Comment:
why are you converting things to linked lists? I tend to avoid them because of concerns about pointer chasing. But I never really looked into the topic in the scala ecosystem, so maybe I'm missing some reasons why they aren't so bad after all?

Reply:
Because certain helper functions in x2cpg require a List. I did not want to change that API in this PR.
But this is a good target for future refactorings I guess.